server/metrics.py

```diff
@@ -103,10 +113,24 @@
     ["game_mode", "game_state"],
 )
 
+game_service_dict_length = Gauge(
```

Comment:
Is this the same as the sum over all reported numbers for `active_games` above?

Reply:
Yep.